README.md

```diff
@@ -55,6 +55,14 @@ and extracting fragments from .wld files.
 # To view fragments
 cargo run -p wld-cli -- explore gfaydark.wld
 
-# Or to extract to raw fragment data files:
+# Extract to raw fragment data files:
 cargo run -p wld-cli -- extract gfaydark.wld destination/
+
+# Extract and create to/from RON:
+cargo run -p wld-cli -- extract --format ron gfaydark.wld gfaydark.ron
+cargo run -p wld-cli -- create --format ron gfaydark.ron gfaydark.wld
+
+# Extract and create to/from JSON:
+cargo run -p wld-cli -- extract --format ron gfaydark.wld gfaydark.json
+cargo run -p wld-cli -- create --format ron gfaydark.json gfaydark.wld
 ```
```

Comment:
This is really useful for playing with the contents of wld files and then loading them into the original client. I've only tested a handful of wld files but _for the most part_ this seems to reliably reproduce valid wld files when extracting and then creating from RON/JSON.